arches_modular_reports/app/views/modular_report.py

```diff
@@ -40,6 +40,18 @@
 )
 
 
+class GraphSlugFromIdView(APIBase):
```

Comment:
why not just get the graph? we already have that endpoint and the extra data over the wire is negligible. That said I really don't feel strongly here so if you _want_ to start down the path of tiny focused endpoints I'm happy to approve.